src/librustdoc/html/render/print_item.rs

```diff
@@ -1604,6 +1606,15 @@ fn document_non_exhaustive(w: &mut Buffer, item: &clean::Item) {
 }
 
 fn document_type_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
+    fn write_size_of_layout(w: &mut Buffer, layout: &Layout, tag_size: u64) {
+        if layout.abi.is_unsized() {
+            write!(w, "(unsized)");
+        } else {
+            let bytes = layout.size.bytes() - tag_size;
```

Comment:
What happens if the `tag_size` is greater than the `layout.size.bytes()`? Or does the `layout.size` for enum variants include the tag size?

Reply:
> Enum-likes with more than one inhabited variant: each variant comes with a discriminant (usually the same as the variant index but the user can assign explicit discriminant values). That discriminant is encoded as a tag on the machine. **The layout of each variant is a struct, and they all have space reserved for the tag.** For enums, the tag is the sole field of the layout.